mmv1/templates/terraform/encoders/bigquery_dataset.go.tmpl

```diff
@@ -0,0 +1,14 @@
+{{/*
+	The license inside this block applies to this file
+	Copyright 2024 Google Inc.
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/ -}}
+obj["accessPolicyVersion"] = 3
```

Comment:
Should `accessPolicyVersion` be added to the file `mmv1/products/bigquery/Dataset.yaml` as a new filed?

Reply:
This part is outdated now